Elisa updating support (#308)
ubvu/open-handbook: guides/data-lifecycle.qmd

General Faculty research support and management guidelines are available in the section [Policies & Regulations](./plan-and-design.qmd#vu-general-policies-and-regulations).

### This website

Questions regarding your data or the information on this website?

> The [RDM Support](mailto:researchdatamanagement@vu.nl?subject=RDM%20website) team can help you with all your question on data management plan, data archiving, data storing or data privacy & security.

Reviewer: ```suggestion
The [RDM Support](mailto:researchdatamanagement@vu.nl?subject=RDM%20website) team can help you with all your questions on data management plans, data storage, data archiving and publishing, and data protection.
```
